<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>672</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>462</width>
    <height>672</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>462</width>
    <height>672</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>규현계산기</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../calculator.png</normaloff>../calculator.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item>
         <widget class="QLineEdit" name="lineEdit_re">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Courier New</family>
            <pointsize>36</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_fo">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Courier</family>
            <pointsize>13</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <property name="spacing">
           <number>1</number>
          </property>
          <item row="0" column="2">
           <widget class="QPushButton" name="pushButton_cl">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>92</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Courier</family>
              <pointsize>13</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(235, 235, 235);
border: 0pt;


</string>
            </property>
            <property name="text">
             <string>C</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="pushButton_ba">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>92</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Courier</family>
              <pointsize>13</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(235, 235, 235);
border: 0pt;


</string>
            </property>
            <property name="text">
             <string>&lt;-</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButton_7">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>92</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Courier</family>
              <pointsize>13</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
border: 0pt;


</string>
            </property>
            <property name="text">
             <string>7</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButton_8">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>92</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Courier</family>
              <pointsize>13</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
border: 0pt;


</string>
            </property>
            <property name="text">
             <string>8</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="pushButton_9">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>92</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Courier</family>
              <pointsize>13</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
border: 0pt;

</string>
            </property>
            <property name="text">
             <string>9</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="pushButton_di">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>92</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Courier</family>
              <pointsize>13</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(235, 235, 235);
border: 0pt;


</string>
            </property>
            <property name="text">
             <string>/</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="pushButton_4">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>92</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Courier</family>
              <pointsize>13</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
border: 0pt;


</string>
            </property>
            <property name="text">
             <string>4</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="pushButton_5">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>92</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Courier</family>
              <pointsize>13</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
border: 0pt;

</string>
            </property>
            <property name="text">
             <string>5</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="pushButton_6">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>92</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Courier</family>
              <pointsize>13</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
border: 0pt;

</string>
            </property>
            <property name="text">
             <string>6</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QPushButton" name="pushButton_mu">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>92</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Courier</family>
              <pointsize>13</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(235, 235, 235);
border: 0pt;


</string>
            </property>
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="pushButton_1">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>92</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Courier</family>
              <pointsize>13</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
border: 0pt;


</string>
            </property>
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="pushButton_2">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>92</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Courier</family>
              <pointsize>13</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
border: 0pt;


</string>
            </property>
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="pushButton_3">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>92</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Courier</family>
              <pointsize>13</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
border: 0pt;


</string>
            </property>
            <property name="text">
             <string>3</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QPushButton" name="pushButton_mi">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>92</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Courier</family>
              <pointsize>13</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(235, 235, 235);
border: 0pt;


</string>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="pushButton_0">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>92</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Courier</family>
              <pointsize>13</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
border: 0pt;


</string>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="pushButton_do">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>92</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Courier</family>
              <pointsize>13</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
border: 0pt;


</string>
            </property>
            <property name="text">
             <string>.</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QPushButton" name="pushButton_eq">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>92</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Courier</family>
              <pointsize>13</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(170, 255, 255);
border: 0pt;


</string>
            </property>
            <property name="text">
             <string>=</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QPushButton" name="pushButton_pl">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>92</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Courier</family>
              <pointsize>13</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(235, 235, 235);
border: 0pt;


</string>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton_gh">
            <property name="minimumSize">
             <size>
              <width>202</width>
              <height>92</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>휴먼편지체</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
border: 0pt;</string>
            </property>
            <property name="text">
             <string>Made by Gyu Hyeon</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
